### PR TITLE
Fix spelling and some docstring improvements

### PR DESCRIPTION
**Pull request by jayvdb**
Signed-off-by: John Vandenberg <jayvdb@gmail.com>

### CODE REVIEW

File: spdx/parsers/tagvaluebuilders.py [hunk 1]
```diff
@@ -1248,13 +1248,13 @@ def add_file_dep(self, doc, value):
 
     def set_file_atrificat_of_project(self, doc, symbol, value):
         """
-        Set a file name, uri or home artificat.
```
Comment (nicoweidner) [inline spdx/parsers/tagvaluebuilders.py:1251]: The artificat strikes again!
(Sorry for the nonsensical comment, but I find this typo hilarious)

**Review by nicoweidner — DISMISSED** [review]
Thanks, much appreciated!

**Comment by nicoweidner** [comment]
Looks like Appveyor is broken for some reason :-/. @licquia @goneall @pombredanne Can any of you do something about this. Sorry about the triple ping, but this currently blocks all PRs, and as far as I can tell, I am lacking permissions to check out what's going on there. The only workaround I can think of is deleting `appveyor.yml`, but that's not really a solution...

**Comment by goneall** [comment]
> Looks like Appveyor is broken for some reason :-/. @licquia @goneall @pombredanne Can any of you do something about this? Sorry about the triple ping, but this currently blocks all PRs, and as far as I can tell, I am lacking permissions to check out what's going on there. The only workaround I can think of is deleting appveyor.yml, but that's not really a solution...

I'm afraid I don't know much about this myself - I don't use Appveyor and do not have a login or permissions to check what's wrong.

@pombredanne - is this something you can help with?

BTW - I noticed this repo is using Travis.  For many of the other SPDX repos we moved to Github actions - something we may want to consider here.

**Review by nicoweidner — APPROVED** [review]

**Pull request merged**